src/buildApplication/processVercelOutput.ts

```diff
@@ -63,6 +67,11 @@ export function processVercelOutput(
 			// `TypeError: Cannot read properties of undefined (reading 'default')`
 			entrypoint: value.replace(/\.rsc\.func\.js$/i, '.func.js'),
 		});
+
+		// Remove the RSC functions from the dist directory as they are not needed.
```

Comment:
```suggestion
		// Remove the RSC functions from the dist directory as they are not needed since we replace them with non-rsc variants for the runtime routing
```
or something along those lines, I think it would be valuable to explain the why here 🙂